duckduckgo/iOS
# Add feedback and broken site form

Task/Issue URL: https://app.asana.com/0/361428290920652/688515420618260
Tech Design URL: N/A

**Description**:
Add a native feedback form for general and broken sites feedback.

**Steps to test this PR**:
TEST 1: Broken site:
1. From the browser page, select the "Report Broken Site"
1. Enter text in the comment box and note that the submit button becomes active
1. Submit the feedback
1. Ensure it appears in https://app.asana.com/0/569473074191537/list
1. Check the url and device details in the Asana task are correct
1. Now delete it from Asana!

TEST 2:  Feedback Message:
1. From the settings page, select "Leave Feedback"
1. Note that the submit button is inactive
1. Enter message text 
1. Submit the feedback
1. Ensure it appears in https://app.asana.com/0/413877547933097/list
1. Check the message and device details in the Asana task are correct
1. Now delete it from Asana!

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
